### DISCUSSION TITLE
How to set the number of GPUs

### DISCUSSION THREAD

**Discussion by DrJohnDale** [nrdg/cloudknot#291] [Q&A] [answered]
Hi,  

I want to run some code on a gpu, i can see how to set a p3 machine but i cannot see how to set the number of GPUs. Is this possible in the current version?

Cheers

**Comment by richford** [comment]
Hi @DrJohnDale, thanks for asking the question. And for bringing to our attention that this is not currently possible in cloudknot. I'll open an issue to expose the [job definition resource requirements](https://docs.aws.amazon.com/batch/latest/userguide/job_definition_parameters.html#ContainerProperties-resourceRequirements) to the user so that one can specify the number of GPUs.

**Comment by richford** [comment, marked as answer]
This issue has been resolved in #295. @DrJohnDale, you can follow [this example notebook](https://github.com/nrdg/cloudknot/blob/master/examples/05_using_gpus.ipynb) to get your multi-GPU jobs up and running. Please feel free to try it out and let us know if it works.
